libs/ledger-live-common/src/exchange/swap/maybeTronEmptyAccount.ts

```diff
@@ -1,13 +1,15 @@
 import { SwapTransactionType } from "./types";
-
+import { TronEmptyAccount } from "@ledgerhq/errors";
 export const maybeTronEmptyAccount = (swapTransaction: SwapTransactionType): Error | undefined => {
   if (
     swapTransaction?.swap?.to?.currency?.type === "TokenCurrency" &&
     swapTransaction?.swap?.to?.currency?.tokenType === "trc20" &&
     swapTransaction?.swap?.to?.parentAccount?.balance.lt(100_000)
   ) {
-    const tezosError = new Error("Cannot swap with an empty TRON account");
-    tezosError.name = "newAddressTRC20";
-    return tezosError;
+    return new TronEmptyAccount(undefined, {
```

Comment:
undefined => "PREVENT_RECEIVING_TRC20_ON_EMPTY_ACCOUNT" par ex.

Reply:
better have a redundant message than a undefined that might cause a reference error later if someone does not pay enough attention